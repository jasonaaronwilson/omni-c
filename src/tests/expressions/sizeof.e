sizeof(uint64_t);