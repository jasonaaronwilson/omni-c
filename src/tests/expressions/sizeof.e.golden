{
    tag = PARSE_NODE_OPERATOR
    operator = sizeof
    right = {
        tag = PARSE_NODE_IDENTIFIER
        token = uint64_t
    }
}

